n2y/main.py

```diff
@@ -95,11 +97,19 @@ def export_markdown(client, raw_rows, options):
             if pandoc_output:
                 markdown = pandoc.write(pandoc_output, format='gfm+tex_math_dollars') \
                     .replace('\r\n', '\n')  # Deal with Windows line endings
-                # reformat equations to have one backslash instead of 2
+
+                # normalize target path and create it if it doesn't exist
+                target = options.target
+                if not target[(len(target)-1)] == '/':
+                    target += '/'
+                    options.target = target
+                if not os.path.exists(target):
```

Comment:
I think it's cleaner (and avoids a small race condition which can occur if the directory is created in between checking if it exists and creating it) if you use `exists_ok` instead of checking if the path exists and then creating it.